rllab/dynamics_randomization/test_dynamics_rand.py

```diff
@@ -0,0 +1,69 @@
+#!/usr/bin/env python3
+"""
+Benchmark model mutation for dynamics randomization
+"""
+import os
+import os.path as osp
+import xml.etree.ElementTree as ET
+
+import numpy as np
+from mujoco_py import load_model_from_xml
+from mujoco_py import MjSim
+from mujoco_py import MjViewer
+
+from rllab.dynamics_randomization import VariationsList
+
+#Execute at the root of rllab
+MUJOCO_PY_PATH = os.getcwd()
+TOSSER_XML = osp.join(MUJOCO_PY_PATH,
+                      "rllab/dynamics_randomization/tosser.xml")
+
+# Load original model text into memory
+tosser = ET.parse(TOSSER_XML)
+
+var_list = VariationsList().\
+    add_variation(".//motor[@name='a1']", "gear", "coefficient", "uniform", (0.5, 1.5)).\
```

Comment:
A [fluent interface](https://en.wikipedia.org/wiki/Fluent_interface) would be more like
```python
variations = Variations()
variations.randomize()
          .attribute("gear")
          .at_xpath(".//motor[@name='a1']")
          .with_method(Variations.COEFFICIENT)
          .sampled_from(Variations.UNIFORM)
          .with_range(0.5, 1.5)
```

Reply:
Ok, that's what @CatherineSue suggested but I thought it's a little verbose, although more understandable.
So, Variations contains a list of variation objects right?
And if that is the case, is the randomize() method used to create a new variation object?

Reply:
well they contain some way of storing variations. a list of objects seems fine. randomize() create a new internal record of a variation.

fluent interfaces are verbose but easy to read. yes, you could use a very long list of arguments, but what if the relevant arguments change based on other arguments?

for instance, if i want a uniform distribution i need a range=(min,max) argument, but if I use a guassian i need a (mean, std) argument. it gets messy very quickly.

Reply:
Got it. @ryanjulian, please review the new implementation. Also, me and @CatherineSue discussed the implementation of the wrapper for the randomized environment and work is in progress on that side as well.